database/liquibase/changelogs/tables/users.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>   
+<databaseChangeLog
+   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xmlns:pro="http://www.liquibase.org/xml/ns/pro"
+   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd
+      http://www.liquibase.org/xml/ns/pro 
+      http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd">
+   <changeSet author="stankowd" id="1">
+      <createTable tableName="user" schemaName="users">
+        <column name="id" type="VARCHAR(255)">
+            <constraints nullable="false"/>
+        </column>
+        <column name="alias" type="VARCHAR(255)"/>
+        <column name="name" type="VARCHAR(255)"/>
+        <column name="surname" type="VARCHAR(255)"/>
+        <column name="birth_date" type="DATE"/>
+      </createTable>
+      <addPrimaryKey schemaName="users" tableName="user" columnNames="id" constraintName="user_primary_key_constraint"/>
+      <createTable tableName="user_credentials" schemaName="users">
+        <column name="password_hash" type="BLOB"/>
+        <column name="email" type="VARCHAR(255)"/>
+        <column name="user_id" type="VARCHAR(255)"/>
+      </createTable>
+      <!-- <addForeignKey baseTableSchemaName="users" baseTableName="user_credentials" baseColumnNames="user_id" constraintName="user_credentials_foreign_key_constraint"
```

Comment:
?

Reply:
also why aren't the relationships mapped? shouldn't credentials have a key? if it's one to one we should probably make it one table @bastyje - wdyt?